content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,68 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updates to existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing resource - for example, a ConfigMap -
+using an HTTP PUT. For a PUT request, it is the client's responsibility to specify
+the `resourceVersion` (taking this from the object being updated). Kubernetes uses
+that `resourceVersion` information so that the API server can detect lost updates
+and reject requests made by a client that is out of date with the cluster.
+In the event that the resource has changed (the `resourceVersion` the clienti
+provided is stale), the API server returns a `409 Conflict` error response.
+
+Instead of sending a PUT request, the client can send an instruction to the API
+server to **patch** an existing resource. A **patch** is appropriate if the
+change that the client wants to make isn't conditional on the existing data.
+Clients that need effective detection of lost updates should consider using
+HTTP PUT, and retrying in case of conflict.
```

Comment:
Two additional things:
1. You can have conditions in your json-patch to validate some things are consistent when the patch is applied (or applying will fail with a 40x bad request)
2. You can also enforce a specific resource version in the patch, which will give you the PUT guarantees without having to send the entire object.

Reply:
@apelisse Do we already have existing docs that show how to set a condition for a patch?

Reply:
I don't think we have one. I know this because I tried and also now because I know how this works internally. That's partly why I wanted to mention it here :-)

Reply:
I reworded, mentioning that (in outline) under `application/json-patch+json`.